code/modules/power/singularity/narsie.dm

```diff
@@ -75,6 +75,7 @@
 	INVOKE_ASYNC(src, .proc/begin_the_end)
 
 /obj/singularity/narsie/large/cult/proc/begin_the_end()
+	to_chat(world, "<BR><BR><BR><span class='big bold'>The round has ended.</span>")
```

Comment:
This is not how round ends work. If you look lower you'll see cult_ending_helper which causes the round to actually end

Reply:
see line 4 of PR

> Can't just move the round-end for reasons that alexkar knows and not me